financial-templates-lib/ExpiringMultiPartyClient.js

```diff
@@ -209,6 +213,10 @@ class ExpiringMultiPartyClient {
             ]),
       []
     );
+    Logger.debug({
```

Comment:
Un related to PR but added to make consistent and cleaner

Reply:
Don't we already emit a similar log in the `update()` function?

Reply:
ye, you are right actually. Have removed it.